src/Adapter/ContainerFinder.php

```diff
@@ -74,6 +75,12 @@ public function getContainer()
             return $container;
         }
 
-        throw new ContainerNotFoundException('Kernel Container is not available');
+        try {
+            $container = CustomContainerBuilder::getContainer('front', _PS_MODE_DEV_);
```

Comment:
Actually, the $container variable is always set, it never fails...

And I think it could be the reason for the broken unit and integration test